test/test_torch.py

```diff
@@ -7237,6 +7237,9 @@ def run_test(p):
         run_test([10, 20, 30, 5])
         run_test([15, 5, 10, 20, 25])
 
+        with self.assertRaisesRegex(RuntimeError, "chain_matmul: Expected one or more matrices"):
```

Comment:
Have updated the existing test here. To verify the behaviour.

Reply:
https://github.com/pytorch/pytorch/blob/5fb52443abe463dc8c33b4e20ceb8cd98b0d77bc/test/test_torch.py#L7224-L7241